reference/locations.v3.yml

```diff
@@ -1731,6 +1735,38 @@ components:
         enum:
           - asc
           - desc
+    date_created_min:
+      name: 'date_created:min'
+      description: |
+        'Query parameter that lets you filter by the minimum date created, for example, `2024-05-14T09:34:00Z`. Returns metafields created after this date.'
```

Comment:
maybe `Query parameter that lets you filter by the date created which is before provided one`

FYI: it works like `>=`

Reply:
no, just if I would be a api user I would have no idea what `minimum date created` is. What I am suggesting is to say that this `min` means that the date provided in the filter will return metafields with the dates which are `after` the one used in filter. 
But if we usually just write minimum I am fine with that as well

Reply:
Hi @bc-rafsierk , How does this sound? Is this clear?
Query parameter that lets you filter by the minimum date created. The date provided in the filter will return metafields after this date. Example format = 2024-05-14T09:34:00Z.